### ISSUE TITLE
time grains broken in sqlite

### ISSUE THREAD

**Issue by xrmx** [apache/incubator-superset#4830]
Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if any
- [ ] I have reproduced the issue with at least the latest released version of superset
- [x] I have checked the issue tracker for the same issue and I haven't found one similar

### Superset version

master from ~1 month ago 97afcd5809269e885a0dde349081802a427c5674

### Expected results

hour, week and month time grains work fine

### Actual results

The result is that hour time grains display an empty chart, while week and month fallback to the day as time grain.

Problem is that sqlite time grains use strftime and thus the `%` that gets escaped breaking the query:

```
SELECT DATETIME(STRFTIME('%%Y-%%m-%%dT%%H:00:00', ts)) AS __timestamp,
       SUM(prezzo) AS sum__prezzo
FROM prova12
WHERE ts >= '2017-09-15 00:00:00.000000'
  AND ts <= '2018-04-15 12:14:41.000000'
GROUP BY DATETIME(STRFTIME('%%Y-%%m-%%dT%%H:00:00', ts))
ORDER BY sum__prezzo DESC
LIMIT 50000
OFFSET 0;
```

### Steps to reproduce

On a Time series line chart querying a sqlite database use hour, week or month as time grain.

**Comment by xrmx** [comment]
Poking a bit with the code this looks added in get_timestamp_expression() by the literal_column call https://github.com/apache/incubator-superset/blob/master/superset/connectors/sqla/models.py#L135

**Comment by xrmx** [comment]
Fixed in 3f48c005df985888f19d46d00cac0f35ec98466d. Thanks!
